### ISSUE TITLE
template is wrong for default nats properties bosh 256.2

### ISSUE THREAD

**Issue by haydonryan** [enaml-ops/omg-cli#10]
```
./omg-linux azure \
  --azure-vnet boshvnet-crp \
  --azure-subnet Bosh \
  --azure-subscription-id <Redacted> \
  --azure-tenant-id <Redacted>  \
  --azure-client-id <Redacted> \
  --azure-client-secret amazon4ever \
  --azure-resource-group hrcentral \
  --azure-storage-account <Redacted> \
  --azure-security-group nsg-bosh \
  --azure-ssh-pub-key-path bosh-privatekey.pub \
  --azure-ssh-user vcap \
  --azure-environment AzureCloud \
  --azure-private-key-path bosh-privatekey \
  --cpi-name cpi \
  --bosh-private-ip 10.0.0.5
```

Generates job properties:

```
nats:
  user:
  password:
  listen_address: 127.0.0.1
```

should be 

```
nats:
  user:
  password:
  address: 127.0.0.1
```

**Comment by calebwashburn** [comment]
Latest version of omg-cli generates the following properties.  Please validate on latest draft release.

return &director.Nats{
        User:     "nats",
        Password: s.NatsPassword,
        Address:  "127.0.0.1",
    }

**Comment by calebwashburn** [comment]
Fixed with release v0.0.10
